tests/tests_integration/test_api/test_transformations/test_transformations.py

```diff
@@ -3,14 +3,33 @@
 
 import pytest
 
-from cognite.client.data_classes import Transformation, TransformationDestination, TransformationUpdate
+from cognite.client.data_classes import DataSet, Transformation, TransformationDestination, TransformationUpdate
 
 
 @pytest.fixture
-def new_transformation(cognite_client):
+def new_datasets(cognite_client):
+    prefix = "".join(random.choice(string.ascii_letters) for i in range(6))
+    ds_ext_id1 = f"{prefix}-transformation-ds"
+    ds_ext_id2 = f"{prefix}-transformation-ds2"
+    ds1 = cognite_client.data_sets.retrieve(external_id=ds_ext_id1)
+    ds2 = cognite_client.data_sets.retrieve(external_id=ds_ext_id2)
+    if not ds1:
+        data_set1 = DataSet(name=ds_ext_id1, external_id=ds_ext_id1)
+        ds1 = cognite_client.data_sets.create(data_set1)
+    if not ds2:
+        data_set2 = DataSet(name=ds_ext_id2, external_id=ds_ext_id2)
+        ds2 = cognite_client.data_sets.create(data_set2)
+    yield ds1, ds2
```

Comment:
A better alternative is to add the api to access control tests, where we test other apis for a thorough dataset capability test

Reply:
ah no that's actually safe-smoke. please ignore the above comment